src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/profiledetails/ProfileDetailsResource.kt

```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.hmpps.nomisprisonerapi.prisonperson.profiledetails
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.profiledetails
```

Comment:
The main change is to move profile details out of the prisonperson package and up to the root package. This is because profile details will span multiple data domains so shouldn't be project related.